static/js/overlays.js

```diff
@@ -102,6 +102,8 @@ exports.open_modal = function (name) {
     blueslip.debug('open modal: ' + name);
 
     $("#" + name).modal("show").attr("aria-hidden", false);
+    // Remove previous alert messsages from modal, if exists.
+    $("#" + name).find(".alert").hide();
```

Comment:
When we open the overlay modal, then it shows previous alert messages. This code is added it to remove it, if it exists.